beta/examples/tensorflow/object_detection/README.md

```diff
@@ -151,4 +151,5 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
 |RetinaNet|Sparsity 50% (Magnitude)|COCO2017|33.13|[retinanet_coco_magnitude_sparsity.json](configs/sparsity/retinanet_coco_magnitude_sparsity.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_sparsity_50.tar.gz)|
 |RetinaNet|Filter Pruning 40%|COCO2017|32.7|[retinanet_coco_pruning.json](configs/pruning/retinanet_coco_pruning.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/retinanet_pruning_40.tar.gz)|
 |YOLOv4|None|COCO2017|47.04|[yolo_v4_coco.json](configs/yolo_v4_coco.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/yolo_v4.tar.gz)|
+|YOLOv4|INT8 w:sym,per-channel a:asym,per-tensor |COCO2017|46.20|[yolo_v4_coco_int8.json](configs/quantization/yolo_v4_coco_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/yolo_v4_int8_w_sym_c_a_asym_t.tar.gz)|
```

Comment:
I would like to ask you to change the checkpoint name. We have already quantized models with per-channel granularity and usually we `ch` alias for this.
```suggestion
|YOLOv4|INT8 w:sym,per-channel a:asym,per-tensor |COCO2017|46.20|[yolo_v4_coco_int8.json](configs/quantization/yolo_v4_coco_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/yolo_v4_int8_w_sym_ch_a_asym_t.tar.gz)|
```